src/server/frontend_wayland/wl_keyboard.cpp

```diff
@@ -49,10 +49,10 @@ void mf::WlKeyboard::handle_event(MirInputEvent const* event, WlSurface& surface
     if (!focused_surface.is(surface))
     {
         log_warning(
```

Comment:
Shouldn't this be a `log_error` now? It happening would be an error in our WM logic, no?

Reply:
Yes, but it should be `fatal_error()`